### PR TITLE
CFD-374: Group approval workflow - update form

### PR DESCRIPTION
**Pull request by solker**
#544

### CODE REVIEW

**Comment by solker** [comment]
>When the request group form is submitted, a group is created but only site administrators will have access to it.

Will be implemented in next PR #547

File: capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 1]
```diff
@@ -68,3 +68,39 @@ function c4m_content_group_preprocess_block(&$variables) {
...
+ * Implements hook_form_FORM_ID_alter().
+ * Hide fields for authenticated users and set submit function.
+ *
+ * @param $form
+ * @param $form_state
+ * @param $form_id
+ */
+function c4m_content_group_form_group_node_form_alter(&$form, &$form_state, $form_id) {
+  global $user;
+  if (!in_array('anonymous user', $user->roles)
```
Comment (HelenaEksler) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: Use ``if (!$account->uid)`` to check if user is anonymous.
Use ``user_access('administer site', $account)`` to check if user is administrator.

**Comment by HelenaEksler** [comment]
@solker Please, add test to check that user doesn't see fields and to check redirection and message appearing.

**Comment by nader77** [comment]
@HelenaEksler PR is ready.

File: capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 2]
```diff
@@ -68,3 +68,36 @@ function c4m_content_group_preprocess_block(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Hide fields for authenticated users and set submit function.
+ */
+function c4m_content_group_form_group_node_form_alter(&$form, &$form_state, $form_id) {
```
Comment (solker) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: remove ``$form_id``

Comment (HelenaEksler) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: @solker Remove it in some of yours future PRs, ok?

**Pull request merged**